frontend/components/receipts/transfer/enriched-transfer-view.tsx

```diff
@@ -0,0 +1,36 @@
+import { formatNumber, toBigNum } from '@vegaprotocol/utils'
+import { PriceWithSymbol } from '../utils/string-amounts/price-with-symbol.tsx'
+import { VegaKey } from '../../keys/vega-key'
+import { useWalletStore } from '../../../stores/wallets.ts'
+import { ReceiptComponentProps } from '../receipts'
+import { VegaAsset } from '../../../types/rest-api.ts'
+
+export const locators = {
+  enrichedSection: 'enriched-section'
+}
+
+interface EnrichedTransferViewProps extends ReceiptComponentProps {
+  assetInfo: VegaAsset | undefined
+}
+
+export const EnrichedTransferView = ({ transaction, assetInfo }: EnrichedTransferViewProps) => {
+  const { getKeyInfo } = useWalletStore()
```

Comment:
Don't use zustand stores without an accessor, triggers a render on every state update
```suggestion
  const { getKeyInfo } = useWalletStore(state => ({ getKeyInfo: state.getKeyInfo }))
```